src/pages/signin/LoginForm/LoginFormNarrow.js

```diff
@@ -102,17 +103,7 @@ class LoginFormNarrow extends React.Component {
                         source={welcomeScreenshot}
                     />
                 </View>
-
-                <View>
-                    <Text style={[styles.textLabel, styles.textStrong, styles.mb1]}>
-                        With Expensify.cash, chat and payments are the same thing.
-                    </Text>
-                    <Text style={[styles.textLabel]}>
-                        Money talks. And now that chat and payments are in one place, it&apos;s also easy.
-                        {' '}
-                        Your payments get to you as fast as you can get your point across.
-                    </Text>
-                </View>
+                <WelcomeText isSmallText />
```

Comment:
Doesn't look like this component has a default prop / handling for `isSmallText`